### ISSUE TITLE
v4: Remame --timeout option (and associated EnginePackageSettings)

### ISSUE THREAD

**Issue by ChrisMaddock** [nunit/nunit-console#890]
To permit https://github.com/nunit/nunit-console/issues/437. --timeout is ambiguous as to whether it refers to the timeout of the full test run, or an individual test. I suggest renaming the option in v4: potentially `--defaultTestTimeout`?

**Comment by CharliePoole** [comment]
Yes. i'd look at the internal names too, including those used as TestPackage settings.

**Comment by CharliePoole** [comment]
@jnm2 @mikkelbu @rprouse 

I lean toward "TestCase" rather than "Test" but --defaultTestCaseTimeout is rather long. What about just --testCaseTimeout?
